packages/app/src/debug/empty/DebugNoProject.vue

```diff
@@ -3,7 +3,6 @@
     :title="t('debugPage.emptyStates.debugDirectlyInCypress')"
     :description="t('debugPage.emptyStates.reviewRerunAndDebug')"
     :example-test-name="t('debugPage.emptyStates.noProjectTestMessage')"
-    :slideshow-campaign="DEBUG_SLIDESHOW.campaigns.connectProject"
     :help-link-sr-text="t('debugPage.emptyStates.learnAboutProjectSetupSrText')"
     help-link-href="https://on.cypress.io/adding-new-project"
   >
```

Comment:
Sorry if I was not clear in the issue, but this `DebugNoProject.vue` should be just like the `DebugNotLoggedIn.vue` and show the promo.  The only thing that changes is the `CloudConnectButton` should automatically update its text to show "Connect a Cypress Cloud project"